contributing-guides/maintainers-guide.md

```diff
@@ -56,17 +56,18 @@ as a guideline for current and future maintainers.
 
 - PRs should be merged once they
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
-  (2) have the **review comments addressed**,
-  (3) get **approved reviews by two maintainers**, (the second maintainer can merge immediately after approving) and
+  (2) have all the **review comments addressed and resolved**,
+  (3) get **approved reviews by at least two maintainers**, (the last maintainer can merge immediately after approving) and
   (4) have been open for at least **24 hours** unless the changes are trivial
 
-- If a PR fails to get a review from a second maintainer after a few days,
+- If a PR fails to get a review from a second maintainer after a week,
   the first maintainer should ping others for review. If it still lingers around
   for **over a week without a second maintainer’s approval**,
-  the first maintainer can go ahead and merge it.
+  the first maintainer can go ahead and merge it while dismissing all reviewers who
+  didn't reviewed the PR.
```

Comment:
```suggestion
  the first maintainer can go ahead and merge it.
```

I don't see the purpose of dismissing, we don't need anyone's opinion (who was requested) before merging, so it just adds more work for the person who merges it and prohibits maintainers, who aren't as active, from reviewing PRs which may have been important however were merged because they had enough reviews.

Reply:
Perhaps what is actually mean here is to dismiss *requests for review*?